src/piral-build.test.ts

```diff
@@ -7,63 +7,192 @@ runTests('piral-build', ({ test, setup }) => {
     await ctx.run(`npm i emojis-list@3.0.0`);
   });
 
+  test('release-standard-template', 'can produce a release build', ['build.piral'], async (ctx) => {
+    await ctx.run(`npx piral build --type release`);
+
+    await ctx.assertFiles({
+      'dist/release/index.html'(content: string) {
+        expect(content).not.toBe('');
+      },
+    });
+  });
+
+  test('emulator-standard-template', 'can produce an emulator build', ['build.piral'], async (ctx) => {
+    await ctx.setFiles({
+      'package.json'(content: string) {
+        const packageJson = JSON.parse(content);
+        packageJson.name = 'app-shell';
+        return JSON.stringify(packageJson, undefined, 2);
+      },
+    });
+
+    await ctx.run(`npx piral build --type emulator`);
+
+    await ctx.assertFiles({
+      'dist/emulator/*.tgz'(files: Array<string>) {
+        expect(files).toHaveLength(1);
+        ctx.setRef('emulator', files[0]);
+      },
+    });
+  });
+
+  test('emulator-sources-standard-template', 'can produce emulator sources build', ['build.piral'], async (ctx) => {
+    await ctx.run(`npx piral build --type emulator-sources`);
+
+    await ctx.assertFiles({
+      'package-lock.json': true,
+      'package.json'(content: string) {
+        expect(content).toContain('"pilets": {');
+      },
+      'node_modules/piral-cli/package.json': true,
+      'emulator/package.json': true,
+      'emulator/app/index.html'(content: string) {
+        expect(content).not.toBe('');
+        expect(content).toContain('<title>My Piral Instance</title>');
+        expect(content).toContain('<script defer src="/index.b68111.js"></script>');
+      },
+    });
+  });
+
+  test(
+    'some-invalid-type-standard-template',
+    'can not produce build with invalid type',
+    ['build.piral'],
+    async (ctx) => {
+      try {
+        await ctx.run(`npx piral build --type some-invalid-type-standard-template`);
```

Comment:
Should error here - so add a check after this line to verify that we successfully fall into the empty `catch` block.